playbooks/satellite/roles/capsule/tasks/main.yaml

```diff
@@ -73,13 +73,24 @@
       name="{{ capsule_installer_pkg }}"
       state=present
 
-  # Prepare for capsule installation
+  # Prepare for 6.2  capsule installation
   - name: "Run capsule-certs-generate on Satellite"
     shell:
       "capsule-certs-generate --capsule-fqdn {{ item }} --certs-tar /var/www/html/pub/{{ item }}-certs.tar >/var/www/html/pub/{{ item }}-out.raw"
     delegate_to: "{{ groups['satellite6']|first }}"
     run_once: true
     with_items: "{{ groups['capsules'] }}"
+    when: "sat_version == '6.2'"
+
+  # Prepare for 6.3  capsule installation
+  # NOTE: The change of --capsule-fqdn to --foreman-proxy-fqdn for Sat6.3
+  - name: "Run capsule-certs-generate on Satellite"
+    shell:
+      "capsule-certs-generate --foreman-proxy-fqdn {{ item }} --certs-tar /var/www/html/pub/{{ item }}-certs.tar >/var/www/html/pub/{{ item }}-out.raw"
```

Comment:
@kbidarkar, isn't this more a bug which should be fixed? Tool have "capsule" in name, but option uses "foreman-proxy"? Looks like a clear bug to me.

Reply:
While I faced this issue, there was already a bug open for it, so I decided just make it work till the bug was fixed. We can revert it , if this is no longer needed. So, raising a bug was not required.

Reply:
This explains all and it seems is not a branding issue , https://bugzilla.redhat.com/show_bug.cgi?id=1470655#c11